developersociety/commonslibrary
# Rebrand

Cards:

amend email addreses
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12736

Directory intro copy changes https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12943

Update Welcome copy https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12738

Update login copy
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12740

Update site header copy
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12737

Update membership policy
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12739

Logo swap and favicon addition
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12733

Resource upload copy changes 
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12987

To test:
This is mainly copy changes with a few assets swapped and added. 

The commits on this branch should map closely to the order of the cards linked above.

Work through these and check all requested changes have been made/ the copy matches that on the cards.
